### PR TITLE
add APIReference-MessengerContext

### PR DESCRIPTION
**Pull request by chentsulin**

### CODE REVIEW

File: content/docs/APIReference-MessengerContext.md [hunk 1]
```diff
@@ -0,0 +1,997 @@
...
+
+* `UPDATE` as default
+* `RESPONSE` using `{ messaging_type: 'RESPONSE' }` options
+* `MESSAGE_TAG` using `{ tag: 'ANY_TAG' }` options
+
+<br />
+
+<a id="content-types" />
+
+### Content Types - [Content types](https://developers.facebook.com/docs/messenger-platform/send-api-reference/contenttypes)
```
Comment (tw0517tw) [inline content/docs/APIReference-MessengerContext.md]: Maybe we should call this `message types`?
Also the doc link has been redirected, need to check if it's redirected to right place as we want.

File: content/docs/APIReference-MessengerContext.md [hunk 2]
```diff
@@ -0,0 +1,997 @@
...
+date: "2017-12-04"
+---
+
+* [Content Types](#content-types)
+* [Templates](#templates)
+* [Quick Replies](#quick-replies)
+* [Sender Actions](#sender-actions)
+* [Attachment Upload API](#attachment-upload-api)
+* [Tags](#tags)
+* [Message Batching](#message-batching)
```
Comment (tw0517tw) [inline content/docs/APIReference-MessengerContext.md]: `Attachment Upload API`
`Tags`
`Message Batching`

these 3 links is not available

File: content/docs/APIReference-MessengerContext.md (hunk 1 above)
Comment (chentsulin) [reply content/docs/APIReference-MessengerContext.md]: or Message Content Types?
https://developers.facebook.com/docs/messenger-platform/send-messages#content_types

Comment (tw0517tw) [reply content/docs/APIReference-MessengerContext.md]: Message Content Types LGTM

**Pull request merged**